script/transpile-web-ui.py

```diff
@@ -168,5 +177,63 @@ def generate_grd(target_include_dir, grd_name, resource_name,
         execute_stdout(args, env)
 
 
+def verify_webpack_srcs(root_gen_dir, data_paths_file, depfile_path,
```

Comment:
Can we add a comment as to what this function is doing as I don't think the name is conveying enough about how they will be verified or what they will be verified against, (and what the webpack srcs are). i.e. is it verifying the data paths or the depfile paths?